src/screens/Profile/Home/index.tsx

```diff
@@ -169,7 +185,11 @@ export default function ProfileHome({navigation}: ProfileScreenProps) {
               <Sections.LinkItem
                 text={t(ProfileTexts.sections.account.linkItems.logout.label)}
                 icon={<ThemeIcon svg={LogOut} />}
-                onPress={() => navigation.navigate('ConsequencesFromLogout')}
+                onPress={() =>
+                  enable_ticketing
+                    ? navigation.navigate('ConsequencesFromLogout')
+                    : logoutAndWipeTokens()
```

Comment:
I have not spent a lot of time adding a loader here. because then we need to decide where we need to add the loader on the profile page. It's a flow that would only happen when we disable ticketing. So I thought it was not worth spending time on. Also, it would be introducing a logic that we practically dont use anywhere.